t/27-consoles-sshIucvconn.t

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/perl
+
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use Test::MockObject;
+use Test::MockModule;
+use Test::Warnings ':report_warnings';
+use Test::Output;
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use consoles::sshIucvconn;  # SUT
+use bmwqemu qw(diag fctwarn);
```

Comment:
Why are you importing the `diag` and `fctwarn` here? you don't use those functions anywhere in the test.
My suggestion was to do this in the module `consoles::sshIucvconn`.
And use `log` instead of `bmwqemu`.

Reply:
Please see #2566 for what I meant

Reply:
Feel free to `git cherry-pick` that commit.

Reply:
e.g.
```
git fetch origin pull/2566/head
git cherry-pick dace5c70
```

Reply:
let me take a look again.  but in any case I might go for the simpliest appoach and do not touch the consoles::sshIucvconn.pm

Reply:
My PR has been merged, so now you only have to rebase and remove the use statement in the test :)

Reply:
rebased and remove the `use bmwqemu qw(diag fctwarn);`